### PR TITLE
Removed deletion of important stock Windows Apps + Allow Windows Hello biometrics

### PR DESCRIPTION
**Pull request by beerisgood**
this would otherwise remove important apps like: new Edge, Paint, Calculator, Photo Viewer, ...

Update: 
Allow Windows Hello biometrics

### CODE REVIEW

File: Scripts/apps.ps1 [hunk 1]
```diff
@@ -17,7 +17,5 @@ Get-WindowsOptionalFeature -Online | Where-Object { $_.FeatureName -like "Intern
 Write-Host "Disable 'Remote Assistance'"
 Get-WindowsCapability -Online | Where-Object { $_.Name -like "App.Support.QuickAssist*" } | Remove-WindowsCapability -Online
 
-Write-Host "Disable Hello Face"
-Get-WindowsCapability -Online | Where-Object { $_.Name -like "Hello.Face*" } | Remove-WindowsCapability -Online
```
Comment (asvc) [inline Scripts/apps.ps1:30]: I would rather keep it disabled as "Hello Face" capabilities are strictly dependent on [hardware implementation](https://nakedsecurity.sophos.com/2018/01/02/windows-hello-face-recognition-spoofed-with-photographs/) which we can't control.

**Review by asvc — CHANGES_REQUESTED** [review]

File: Scripts/apps.ps1 [hunk 2]
```diff
@@ -5,15 +5,6 @@
 # https://github.com/stoptracking/windows10
 #
 
-Write-Host "Remove pre-installed applications..."
```
Comment (asvc) [inline Scripts/apps.ps1:8]: Please provide list of "must have" UWP apps that are being removed. As far as I can see, Edge is still available.

File: Scripts/apps.ps1 (hunk 1 above)
Comment (beerisgood) [reply Scripts/apps.ps1:30]: Hello Face Unlock can be harden:
https://www.tenforums.com/tutorials/101265-enable-enhanced-anti-spoofing-windows-hello-face-authentification.html

Also see the comparison: https://docs.microsoft.com/de-de/windows/security/identity-protection/hello-for-business/hello-biometrics-in-enterprise#gesichtserkennungssensorenfacial-recognition-sensors

File: Scripts/apps.ps1 (hunk 2 above)
Comment (beerisgood) [reply Scripts/apps.ps1:8]: From https://github.com/stoptracking/windows10/pull/8/commits/3cbf238a8092c3d09b1696046952d7dcb73329a8

new (Chromium-)Edge, Paint, Calculator, Photo Viewer, Xbox stuff, ..

User should have control about apps

**Pull request closed without merging**